backend/corpora/common/corpora_config.py

```diff
@@ -11,8 +11,9 @@ def __init__(self, *args, **kwargs):
 
     def get_defaults_template(self):
         template = {"upload_file_formats": ["h5ad"], "upload_max_file_size_gb": 30}
-        if os.getenv("UPLOAD_SFN_ARN"):
-            template["upload_sfn_arn"] = os.getenv("UPLOAD_SFN_ARN")
+        upload_snf_arn = os.getenv("UPLOAD_SFN_ARN")
```

Comment:
rather than call `os.getenv` twice for the name variable, we call it once and store it locally.